operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/config/impl/LegacyConfigurationImpl.java

```diff
@@ -49,15 +49,15 @@ public String getCoreApiBatchTrustStorePassword() {
     }
 
     @Override
-    public long getCoreCosMaxResources() {
+    public long getCoreCosObjectResourceCountThreshold() {
         final String PATH = "fhirServer/bulkdata/cosFileMaxResources";
-        return FHIRConfigHelper.getIntProperty(PATH, DEFAULT_COSFILE_MAX_RESOURCESNUMBER);
+        return FHIRConfigHelper.getIntProperty(PATH, DEFAULT_COS_OBJ_MAX_RESOURCE_COUNT);
     }
 
     @Override
-    public long getCoreCosThresholdSize() {
+    public long getCoreCosObjectSizeThreshold() {
         final String PATH = "fhirServer/bulkdata/cosFileMaxSize";
-        return FHIRConfigHelper.getIntProperty(PATH, DEFAULT_COSFILE_MAX_SIZE);
+        return FHIRConfigHelper.getIntProperty(PATH, DEFAULT_COS_OBJ_MAX_SIZE_MB);
```

Comment:
I just changed their names to match the newly proposed config names...happy to discuss the naming of those config parameters since this is our one chance to get it right.  The constant names obviously matter a lot less...just tried to keep them fairly consistent.

Reply:
right, what I'm saying is by changing to 1000s we've actually changed the meaning of legacy.  
it's not what you did, it's that we really don't support legacy implementations out of the box.

Reply:
ah, ok.  I was wondering about that...did we even have these properties before?  I don't think I ever set them

Reply:
We did have them before... 


fhirServer/bulkdata/cosFileMaxResources | int | The maximum number of FHIR resources per COS file, “-1” means no limit, the default value is 200000
-- | -- | --
fhirServer/bulkdata/cosFileMaxSize | int | The maximum COS file size in bytes, “-1” means no limit, the default value is 209715200 (200M)

Reply:
ok, confirmed we did have this before.  how about we just do this:
`return FHIRConfigHelper.getIntProperty(PATH, DEFAULT_COS_OBJ_MAX_SIZE_MB * 1024 * 1024);`